RELEASE.md

```diff
@@ -1,3 +1,13 @@
+# Release 2.11.1
+
+A patch to replace the deprecated OOB auth flow, and remove deprecated aliases
```

Comment:
For consistency with previous releases, I'd probably skip the summary line here - the fact that it's a patch is implied by the version number, and the rest is just restating what's included in the bullets below.